### ISSUE TITLE
Backend: Secure QR-code generation

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#76]
## User story
1. As a {HR-employee}
2. I want / need {to be sure that no third party can access the QR-code that is created from the Lissi-URl}
3. So that {no third party can initialize the connection/digital identity in the Lissi wallet}

## Acceptance criteria
* found secure tool for creation of QR-code
* documentation of how the process of QR-code generation works
* estimation of how much work it will be to implement this secure method in the backend

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
